src/devices/Hcsr04/samples/README.md

```diff
@@ -12,16 +12,21 @@
 * Trig - GPIO 4
 * Echo - GPIO 17
 
-The fritz diagram above depicts how you should wire your RPi in order to run the program, the resistance for R1 is 1kOhm. (Optional)
+The fritz diagram above depicts how you should wire your RPi in order to run the program, the resistance for R1 is 1kOhm. (Optional).
 
 ## Code
 ```C#
 using (var sonar = new Hcsr04(4, 17))
 {
-    while (true)
+    try
     {
-        Console.WriteLine($"Distance: {sonar.Distance} cm");
-        Thread.Sleep(1000);
+        Console.WriteLine($"Distance: {sonar.Distance.Centimeters} cm");
```

Comment:
we might want to make TryGetDistance public just to avoid this anti-pattern of catching

Reply:
yes, made the Try public and adjusted the sample. Will leave the GetDistance() and Distance public anyway